examples/sample-app/application-template-custombuild.json

```diff
@@ -66,6 +66,7 @@
       "parameters": {
         "output": {
           "to": {
+            "kind": "ImageStream",
```

Comment:
does the default get populated so that API consumers don't have to know that "" means "ImageStream"?